superdesk/publish/transmitters/http_push.py

```diff
@@ -44,7 +44,7 @@ def _transmit(self, queue_item, subscriber):
             response = requests.post(resource_url, data=json.dumps(item), headers=self.headers)
             if response.status_code not in (requests.codes.created, requests.codes.accepted):  # @UndefinedVariable
                 message = 'Error pushing item %s: %s' % (response.status_code, response.text)
-                raise PublishHTTPPushError.httpPushError(Exception(message, destination))
+                raise PublishHTTPPushError.httpPushError(Exception(message, destination, response.status_code))
```

Comment:
shouldn't it be `raise PublishHTTPPushError.httpPushError(Exception(message), destination, response.status_code)` ?